examples/trace-analytics-sample-app/data-prepper-wait-for-os-and-start.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+#
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+
+until [[ $(curl --write-out %{http_code} --output /dev/null --silent --head --fail https://node-0.example.com:9200 -u admin:admin --insecure) == 200 ]]; do
+  echo "Waiting for OpenSearch to be ready"
+  sleep 1
+done
+
+java -jar data-prepper.jar pipelines.yaml data-prepper-config.yaml
```

Comment:
Is there a particular reason you are not using `bin/data-prepper` here? I'd like to use that as much as possible going forward.

Reply:
This sample is pulling latest released image from DockerHub, i.e. Data Prepper 1.5.1, so `bin/data-prepper` doesn't work here yet. 

There's one or two other places like this too. I'm planning to update all those after we publish the 2.0 image.

Reply:
Yes, that's a good point. And I think the approach you are taking is best here. Thanks!

Reply:
Will follow up in #1805